$MirOS: ports/mailnews/pine/patches/patch-imap_Makefile,v 1.2 2005/05/04 23:37:39 tg Exp $
$OpenBSD: patch-Makefile,v 1.5 2003/10/05 22:00:00 jakob Exp $
--- imap/Makefile.orig	Sat Apr 30 20:49:58 2005
+++ imap/Makefile	Fri Oct  7 19:12:53 2005
@@ -180,7 +180,7 @@ SSLTYPE=nopwd
 # 4	(default) IPv4 support only
 # 6	IPv6 and IPv4 support
 
-IP=4
+IP=6
 
 
 # The following extra compilation flags are defined.  None of these flags are
@@ -475,8 +475,6 @@ sslunix sslsco:
 	@echo ++ SSLTYPE=$(SSLTYPE).nopwd
 	@echo +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 	@echo
-	@echo Do you want to continue this build anyway?  Type y or n please:
-	@$(SH) -c 'read x; case "$$x" in y) exit 0;; *) exit 1;; esac'
 
 sslnone:
 	@echo +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
